Applications/Spire/Source/Ui/OverlayPanel.cpp

```diff
@@ -140,10 +140,8 @@ bool OverlayPanel::event(QEvent* event) {
 }
 
 void OverlayPanel::keyPressEvent(QKeyEvent* event) {
-  if(event->key() == Qt::Key_Escape) {
-    if(!m_is_closed) {
-      fade(true);
-    }
+  if(event->key() == Qt::Key_Escape && !m_is_closed) {
```

Comment:
This is even shorter than what I had in mind, but for the early return I meant something like this:

```
if(event->key() == Qt::Key_Escape && !m_is_closed) {
  fade(true);
  return;
}
QWidget::keyPressEvent(event);
```